### ISSUE TITLE
ROSCOE scores - significance of score increases

### ISSUE THREAD

**Issue by SahanaRamnath** [facebookresearch/ParlAI#4941]
@Golovneva 
I've been used ROSCOE scores to evaluate my predicted rationales, and I was wondering how to interpret and compare the scores of two sets of rationales. For example, if one set of rationales has a coherence score of say 0.7, and another has a score of 0.8, would we consider this difference of 0.1 to be significant? What if the scores were even closer, say 0.73 and 0.75?

For clarification: If I were using a metric such as accuracy, I would know exactly what a certain score meant, and I would know how to compare the scores given by two different models. Or, if I were to use the BLEU metric, I know that BLEU increases are more significant when the absolute values of the scores are lesser (an increase from 0.3 to 0.35 is more significant than an increase from 0.7 to 0.75). Are they any similar trends/analyses with the ROSCOE scores?

**Comment by Golovneva** [comment]
That's a great question, and we look into it in the (Analysis section)[https://openreview.net/pdf?id=xYlJRpzZtsY], asking if there is a set thresholds in our scores that could separate "good" chains from "bad". The short answer is no: first, the metric values vary across metric types, second, between rationales. The last one means if we compare Chain_1 with coherence 0.7 and Chain_2 with coherence 0.73, they might look equally good. That said, if we look at distributions of scores in the set, rather than at individual chains, and they look similar but with different mean, we can say that the one with lower mean is worse on that metric. Now if we look at two chains produced for the same context, than it is likely that the one with higher score will be better in that particular error type (that correspond to the metric we are looking at). But if those are two different examples, than this will not hold (see distribution overlaps in Fig 2, Fig 7). Some scores (Info-chain, or faithfulness-step) show large (~>0.1) differences in mean values for datasets with and without errors, but I can't say that we found that, for example, BERT score values could be separable, i.e. we could say that there is a significant difference at all in the example of repetition error.
To sum up, I would recommend to compare mean values across prediction sets rather than individual values per sample. We don't have a recommendations for significance of thresholds, and always confirm the difference with 1) significance testing; 2) manual review for the top-scores. So far I found that the quality of chains if visibly different when mean values differ by ~>0.05.

**Comment by SahanaRamnath** [comment]
Thank you!
